bin/hdfcoinc/pycbc_multiifo_combine_statmap

```diff
@@ -26,25 +26,21 @@ f = h5py.File(args.output_file, "w")
 logging.info('Copying segments and attributes to %s' % args.output_file)
 # Move segments information into the final file - remove some duplication in earlier files
 for fi in files:
-    # Get a list of ifos in the segments of the input data file
-    ifos_in_file = [ifo for ifo in args.ifos if ifo in fi['segments']]
-    # Arbitrarily choose one to copy over the segments for as all are currently the same 
-    ifo_to_get = 'segments/{}'.format(ifos_in_file[0])
-    # Convert list of IFOs into a identifier string
-    ifosstring = ''.join(ifos_in_file)
-    group_to_save = '{}/segments'.format(ifosstring)
-    f[group_to_save + '/end'] = fi[ifo_to_get + '/end'][:]
-    f[group_to_save + '/start'] = fi[ifo_to_get + '/start'][:]
-    # Use this ID string to save the attributes from the input files
+    if len(fi['segments'].keys()) > 1:
+        logging.info('Incorrect segment format.')
+
+    key = fi['segments'].keys()[0]
+    f['segments/%s/end' % key] = fi['segments/%s/end' % key][:]
```

Comment:
It would be good to have a check that none of the detector combinations have been used before (`if key in f['segments']`) and then give an error if they have

I should have included it before, but hadn't thought of it

Reply:
That can only occur if the same file is in the input twice, that should never happen, at least within a workflow. The input is 1 file per detector combination. I do not think this is needed.